app/src/main/res/layout/list_project.xml

```diff
@@ -33,10 +33,11 @@
             android:padding="16dp"
             android:textAppearance="?attr/textAppearanceHeadline6" />
         <com.google.android.material.chip.ChipGroup
-            android:id="@+id/chip_group"
+            android:id="@+id/tags_chip_group"
```

Comment:
Changing name could lead to misunderstanding because it is not longer only for tags, but also for sections.